css/style.css

```diff
@@ -80,4 +80,12 @@ main {
 
 .numberBox {
     font-size: 6rem;
+}
+
+.numberBox:hover {
+    cursor: not-allowed;
+}
+
+.numberBox:hover:not(.clicked){
```

Comment:
if you change the one above, you wont need a not here, which will have better browser support